### ISSUE TITLE
should we use less steps to fine-tuning？

### ISSUE THREAD

**Issue by DavidShu2017** [kenshohara/3D-ResNets-PyTorch#109]
Hello sir
      When I reproduce your experiment using the resnext-101-pretrained model, I found that the accuracy of the results obtained by softmax regression at 40 steps is higher than that in your paper, with the accuracy of top1 reaching 98.678%. So I think, is your result in the paper over-fitting? In addition, in the process of fine-tuning, should there be fewer steps to train?

**Comment by DavidShu2017** [comment]
sorry I am worry... I used top-5 accuracy.
